tests/e2e/helper_test.go

```diff
@@ -94,7 +94,11 @@ func setupDSCICR(name string) *dsciv1.DSCInitialization {
 			},
 			TrustedCABundle: &dsciv1.TrustedCABundleSpec{
 				ManagementState: "Managed",
-				CustomCABundle:  "",
+				CustomCABundle: `-----BEGIN CERTIFICATE-----
+      MIIFVjCCAz6gAwIBAgIUQ+NxE9izWRRdt86M/TX9b7wFjUUwDQYJKoZIhvcNAQEL
+     ...
+      IrrVQJLuM7IjWcmOvFjai57QGfIvWcaMY1q6n6MLsLOaXLoRuBLpDLvPbmyAhykU
+      ------END ------` + "\n",
```

Comment:
I am not sure we need to add `\n` here, since that was the change - to introduce newline if its not in input
cc: @zdtsw correct me if I missed something here

Reply:
@VaishnaviHire I thought the same but if i dont add newline it there, the check here https://github.com/opendatahub-io/opendatahub-operator/blob/f72a8a2dee3acea867c6a9f88cb6f9bacc148c5e/tests/e2e/creation_test.go#L474 fails

Reply:
>I am not sure we need to add \n here, since that was the change - to introduce newline if its not in input

correct, user input data in DSCI should not need to explicility add newline ending